vms/avm/state/state.go

```diff
@@ -389,29 +430,34 @@ func (s *state) Commit() error {
 	return batch.Write()
 }
 
-func (s *state) Abort() {
+func (s *State) Abort() {
 	s.db.Abort()
 }
 
-func (s *state) CommitBatch() (database.Batch, error) {
+func (s *State) CommitBatch() (database.Batch, error) {
 	if err := s.write(); err != nil {
 		return nil, err
 	}
 	return s.db.CommitBatch()
 }
 
-func (s *state) Close() error {
+func (s *State) UTXOs(startingUTXOID ids.ID) iter.Seq2[*avax.UTXO, error] {
+	return s.utxoState.UTXOs(startingUTXOID)
+}
+
+func (s *State) Close() error {
 	return errors.Join(
-		s.utxoDB.Close(),
-		s.txDB.Close(),
-		s.blockIDDB.Close(),
-		s.blockDB.Close(),
+		s.UTXODB.Close(),
+		s.UTXOIndexDB.Close(),
```

Comment:
Previously the `avax` package but it didn't close this db... so this is a behavior change.